src/test/java/mil/dds/anet/test/resources/TaskResourceTest.java

```diff
@@ -384,4 +384,13 @@ private void createTask(Person user) {
     }
   }
 
+  @Test
+  void shouldBeSearchableViaCustomFields() {
+    final var query = TaskSearchQueryInput.builder().withText("red").build();
+    final var searchObjects =
+        withCredentials(adminUser, t -> queryExecutor.taskList(getListFields(FIELDS), query));
+    assertThat(searchObjects).isNotNull();
+    assertThat(searchObjects.getTotalCount()).isGreaterThan(0);
+  }
+
```

Comment:
At least be consistent: either keep the blank line in all test classes you updated (which had a blank line at the end), or remove it everywhere. :smile: